### ISSUE TITLE
[ButtonGroup]: Improve ZeroHeight accessibility documentation

### ISSUE THREAD

**Issue by mart-sessman** [TEDI-Design-System/general#174]
Currently there is no information in tedi [zeroheight](https://www.tedi.ee/1ee8444b7/p/82e9cf-button-group/b/139e70) about which aria tags need to be used for selected ButtonGroup button.
Veera doc says that "aria-selected" needs to be used.

**Comment by mart-sessman** [comment]
as per mozilla.org reference, aria-pressed or aria-current should be used, aria-selected is not a valid role on button element.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-pressed
I'd go with aria-pressed as the docs say that "Adding aria-pressed to an element with a role of button turns the button into a toggle button"
